update.php

```diff
@@ -94,7 +94,7 @@ function locfileToPhp($locContent){
 	
 		$version = $_GET['version'];
 		$urlVar = "UNITDB_FILES_API_URL_FORMAT";
-		$apiUrl = "https://api.faforever.com/featuredMods/0/files/%s";
+		$apiUrl = "https://content.faforever.com/featuredMods/0/files/%s";
```

Comment:
That won't do it. This is the url to the API which gives you actual downloadable urls for these files.
You need to replace the urls in the result from `https://content.faforever.com` to `http://faf-content`